pwa/src/templates/components/ComponentsTemplate.tsx

```diff
@@ -107,31 +130,16 @@ export const ComponentsTemplate: React.FC<ComponentsTemplateProps> = ({ defaultT
         </FormField>
       </form>
 
-      <div className={styles.componentsGrid}>
-        {!getComponents.isLoading ? (
-          filteredComponents.map((component) => (
-            <RichContentCard
-              key={component.id}
-              link={{ label: component.name, href: `/components/${component.id}` }}
-              labelsWithIcon={[
-                { label: _.upperFirst(component.intendedAudience), icon: <HamburgerIcon /> },
-                { label: "Conduction", icon: <HouseIcon /> },
-              ]}
-              tags={[component.developmentStatus, component.softwareType]}
-              contentLinks={[
-                { title: "Repository", subTitle: "Bekijk de repository op GitHub", href: component.isBasedOn },
-              ]}
-            />
-          ))
-        ) : (
-          <>
-            <Skeleton height="250px" />
-            <Skeleton height="250px" />
-          </>
-        )}
-
-        {!filteredComponents.length && !getComponents.isLoading && t("No components found with active filters")}
-      </div>
+      {!getComponents.isLoading ? (
+        <ComponentResultTemplate results={filteredComponents} type={display} />
+      ) : (
+        <>
+          <Skeleton height="250px" />
+          <Skeleton height="250px" />
+        </>
+      )}
+
+      {!filteredComponents.length && !getComponents.isLoading && t("No components found with active filters")}
```

Comment:
The table headers shouldn't be visible at this point:

![image](https://user-images.githubusercontent.com/66552932/177326635-53f7a699-82b8-4674-904e-a9e2ae38a1e9.png)